packages/apollo-server-core/src/requestPipeline.ts

```diff
@@ -385,9 +388,7 @@ export async function processGraphQLRequest<TContext>(
     if (config.formatResponse) {
       const formattedResponse: GraphQLResponse | null = config.formatResponse(
         response,
-        {
-          context: requestContext.context,
-        },
+        requestContext,
```

Comment:
Well that was easy 😄I read the PR and thought - "how's he gonna get out of _that_ breaking change?" Interesting and convenient that this was passed in as an object before!